packages/fxa-auth-server/lib/error.js

```diff
@@ -1018,6 +1018,15 @@ AppError.totpTokenAlreadyExists = () => {
   });
 };
 
+AppError.totpTokenDoesNotExist = () => {
+  return new AppError({
+    code: 400,
+    error: 'Bad Request',
+    errno: ERRNO.TOTP_TOKEN_DOES_NOT_EXIST,
+    message: 'TOTP token does not exist for this account.',
```

Comment:
Should some of the "TOTP token" references in this PR actually be "TOTP secret" to help disambiguate?